### PR TITLE
Ft gbr 32 views serializers cleanup

### PR DESCRIPTION
**Pull request by fablet**

### CODE REVIEW

**Comment by fablet** [comment]
I think the large commit history is because I merged in the updated develop branch after branching off of tallus-residential-cert-api

File: seed/serializers/organizations.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+from rest_framework import serializers
+
+# Local Imports
+from seed.lib.superperms.orgs.models import ROLE_OWNER, OrganizationUser
+from seed.models import Cycle, PropertyView, TaxLotView
+from seed.public.models import INTERNAL, PUBLIC, SharedBuildingField
+from seed.utils.users import _get_js_role
+
+
+# TODO: _dict_org uses CanonicalBuilding; delete or fix
```
Comment (tallus) [inline seed/serializers/organizations.py]: Can you remove these comments if its fixed

File: seed/views/users.py [hunk 2]
```diff
@@ -123,11 +41,13 @@ def validate_request_user(self, pk, request):
             user = User.objects.get(pk=pk)
         except ObjectDoesNotExist:
             return False, JsonResponse(
-                {'status': 'error', 'message': "Could not find user with pk = " + str(pk)},
+                {'status': 'error', 'message':
+                    "Could not find user with pk = " + str(pk)},
```
Comment (tallus) [inline seed/views/users.py]: This should be "Could not find user with pk = {}".format(pk)

File: seed/views/users.py [hunk 3]
```diff
@@ -136,16 +56,19 @@ def validate_request_user(self, pk, request):
...
-        Creates a new SEED user.  One of 'organization_id' or 'org_name' is needed.
+        Creates a new SEED user.  One of 'organization_id' or 'org_name'
+        is needed.
         Sends invitation email to the new user.
         ---
         parameters:
             - name: organization_id
-              description: Organization ID if adding user to an existing organization
+              description: Organization ID if adding user to an existing
+                organization
```
Comment (tallus) [inline seed/views/users.py]: This might need to  be indented 4 spaces to avoid Sphinx errors

**Comment by tallus** [comment]
Minor comments but looks good otherwise

**Pull request closed without merging**